src/modules/ekf2/EKF/ekf.h

```diff
@@ -234,7 +232,7 @@ class Ekf final : public EstimatorInterface
 #endif // CONFIG_EKF2_GRAVITY_FUSION
 
 	// get the state vector at the delayed time horizon
-	const matrix::Vector<float, State::size> &getStateAtFusionHorizonAsVector() const { return _state.vector(); }
+	const matrix::Vector<float, 24> &getStateAtFusionHorizonAsVector() const { return _state.vector(); }
```

Comment:
need to fix that hardcoded 24

Reply:
We also have https://github.com/PX4/PX4-Autopilot/blob/a2aa2ff457fe1d4268ede49348a6938bc0917dbe/src/modules/ekf2/EKF/ekf.h#L87

Reply:
done